### PR TITLE
New_Option_US09: Don't display empty profile fields

### PR DESCRIPTION
**Pull request by Rdidmohamedelhadi**

### CODE REVIEW

File: webapp/portlet/src/main/webapp/profile-contact-information/components/ProfileContactPhone.vue [hunk 1]
```diff
@@ -1,24 +1,27 @@
...
-      class="align-end flex-grow-1 text-truncate text-end skeleton-text skeleton-text-width skeleton-background skeleton-text-height-fine skeleton-border-radius">
-      &nbsp;&nbsp;&nbsp;&nbsp;
-    </div>
-    <div v-else-if="user && user.phones && user.phones.length" class="align-end flex-grow-1 text-truncate text-end">
-      <div v-for="(phone, i) in user.phones" :key="i" :title="phone.phoneNumber" class="text-no-wrap text-truncate">
-        <span class="pr-1 text-capitalize">
-          {{ getLabel(`profileContactInformation.phone.${phone.phoneType}`, phone.phoneType) }}:
-        </span>
-        {{ phone.phoneNumber }}
+  <div>
```
Comment (azayati) [inline webapp/portlet/src/main/webapp/profile-contact-information/components/ProfileContactPhone.vue]: the condition to be added here

File: webapp/portlet/src/main/webapp/profile-contact-information/components/ProfileContactUrls.vue [hunk 2]
```diff
@@ -1,23 +1,26 @@
...
-      v-if="skeleton"
-      class="align-end flex-grow-1 text-truncate text-end skeleton-text skeleton-text-width skeleton-background skeleton-text-height-fine skeleton-border-radius">
-      &nbsp;&nbsp;&nbsp;&nbsp;
-    </div>
-    <div v-else-if="user.urls && user.urls.length" class="align-end flex-grow-1 text-truncate text-end">
-      <div v-for="(url, i) in user.urls" :key="i" class="text-no-wrap text-truncate">
-        <a :href="url.url" :title="url.url" target="_blank" rel="noopener noreferrer">
-          {{ url.url }}
-        </a>
+  <div>
```
Comment (azayati) [inline webapp/portlet/src/main/webapp/profile-contact-information/components/ProfileContactUrls.vue]: the condition to be added here

File: webapp/portlet/src/main/webapp/profile-contact-information/components/ProfileContactIms.vue [hunk 3]
```diff
@@ -1,28 +1,31 @@
...
-      class="align-start text-no-wrap font-weight-bold mr-3">
-      {{ $t('profileContactInformation.ims') }}
-    </div>
-    <div
-      v-if="skeleton"
-      class="align-end flex-grow-1 text-truncate text-end skeleton-text skeleton-text-width skeleton-background skeleton-text-height-fine skeleton-border-radius">
-      &nbsp;&nbsp;&nbsp;&nbsp;
-    </div>
-    <div v-else-if="user && user.ims && user.ims.length" class="align-end flex-grow-1 text-truncate text-end">
+  <div>
```
Comment (azayati) [inline webapp/portlet/src/main/webapp/profile-contact-information/components/ProfileContactIms.vue]: the condition to be added here

**Review by azayati — CHANGES_REQUESTED** [review]
See review

**Review by azayati — DISMISSED** [review]
LGTM

**Review by ghost — DISMISSED** [review]
No merge please until code freeze ends

**Review by azayati — DISMISSED** [review]
LGTM

**Review by ghost — DISMISSED** [review]
As we said before there is no merge until the code freeze finishes ... Release is running.

**Review by azayati — DISMISSED** [review]
LGTM

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**